garage/experiment/local_tf_runner.py

```diff
@@ -120,19 +120,20 @@ def setup(self, algo, env, sampler_cls=None, sampler_args=None):
             else:
                 sampler_cls = OffPolicyVectorizedSampler
 
-        self.sampler = sampler_cls(algo, **sampler_args)
+        self.sampler = sampler_cls(algo, env, **sampler_args)
 
         self.initialize_tf_vars()
         self.has_setup = True
 
     def initialize_tf_vars(self):
         """Initialize all uninitialized variables in session."""
-        self.sess.run(
-            tf.variables_initializer([
-                v for v in tf.global_variables()
-                if v.name.split(':')[0] in str(
-                    self.sess.run(tf.report_uninitialized_variables()))
-            ]))
+        with tf.name_scope("auto-initialized"):
```

Comment:
hmm what prompted this change?

Reply:
The current tf graph looks like this: (many report_uninitialized_variables)
![image](https://user-images.githubusercontent.com/8605658/53993779-2db47380-40e5-11e9-82db-1a86d0270c29.png)

Reply:
ah i see. can you change the scope name to `'initialize_tf_vars'`?

That will make it easy for someone find what code produced this when they see it in TensorBoard.

Reply:
also, can  you show the "after" TensorBoard corresponding to Chang's "before"?

Reply:
<img src="https://user-images.githubusercontent.com/8409272/54252202-2c30e400-4507-11e9-8e0d-df289ea41ca8.png" height=500px>

Reply:
looks great.